config_example.yml

```diff
@@ -4,6 +4,7 @@ github_organization: ""
 # Cloud nine connection infos
 cloud_nine_region: ""
 cloud_nine_tenant: ""
+cloud_nine_ssh_keys: ""
```

Comment:
I'm not sure about naming this var cloud_nine... where it could just be ssh_keys